port/cpl_vsi.h

```diff
@@ -719,6 +719,13 @@ void CPL_DLL VSIFreeFilesystemPluginCallbacksStruct(
 int CPL_DLL VSIInstallPluginHandler(
     const char *pszPrefix, const VSIFilesystemPluginCallbacksStruct *poCb);
 
+/**
+ * Unregister a handler previously installed with VSIInstallPluginHandler() on
+ * the given prefix.
+ * @since GDAL 3.9
```

Comment:
```suggestion
 * the given prefix.
 * Note: it is generally unsafe to remove a handler while there are still file handles
 * opened managed by that handler. It is the responsibility of the caller to ensure
 * that it calls this function in a situation where it is safe to do so.
 * @since GDAL 3.9
```

Reply:
Also: Is the `@since` versioning correct?

Reply:
> s the `@since` versioning correct?

yes